src/Controller/ExerciseController.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Exercise;
+use App\Form\ExerciseType;
+use App\Repository\ExerciseRepository;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @Route("/exercise")
+ */
+class ExerciseController extends BaseController
+{
+    /**
+     * @Route("/", name="exercise_index", methods={"GET"})
+     */
+    public function index(ExerciseRepository $exerciseRepository): Response
+    {
+        return $this->render('exercise/index.html.twig', [
+            'exercises' => $exerciseRepository->findAll(),
+        ]);
+    }
+
+    /**
+     * @Route("/new", name="exercise_new", methods={"GET","POST"})
+     */
+    public function new(Request $request): Response
+    {
+        $exercise = new Exercise();
+        $form = $this->createForm(ExerciseType::class, $exercise);
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            if ($form->get('intervals')->getData() == null) {
+                $exercise->setIntervals(1);
+            }
```

Comment:
@daria-sieroshtan 
This does belong to the controller.
You can use form event, data transformer, or doctrine event to handle this properly.
You should also have DB constraint if you rely on this thing not being null